web-frontend/src/mirage/server.ts

```diff
@@ -35,6 +38,13 @@ function mockServer(environment = ENV.DEV) {
 
         return schema.find('account', nickname);
       });
+
+      this.put('accounts', (schema) => {
+        return schema.create('account', {
+          nickname: 'imported',
+          candidateBalance: '117',
+        });
+      });
```

Comment:
the wallet API is designed like this. this PUT on /accounts is only wake uping the wails prompt to import an account. nothing to update, it will create an new account.

there is no param, this is why I create an new one.

Reply:
Ok! Fair enough then! Mirage is a mirror of the server. If is implemented like that... there's nothing we can mock different then! :)